docs/source/api/spots/index.rst

```diff
@@ -38,6 +38,18 @@ Spot Detectors can be imported using ``starfish.spots.DetectSpots``, which regis
 .. autoclass:: starfish.spots.DetectSpots
     :members:
 
+NOTE: Starfish is embarking on a SpotFinding data structures refactor see `Spot Finding Refactor Plan`_
+DetectSpots will be replaced by the FindSpots module documented below.
+
+.. _`Spot Finding Refactor Plan`: https://github.com/spacetx/starfish/issues/1514
+
+.. code-block:: python
+
+    from starfish.spots import FindSpots
+
+.. autoclass:: starfish.spots.FindSpots
```

Comment:
Isn't FindSpots a module?  I think you want `automodule` here to recursively pull in all the implementations in `FindSpots`, and the weird logic to import in all the submodules of `FindSpots`.